.platform.app.yaml

```diff
@@ -123,5 +123,5 @@ web:
 # The configuration of scheduled execution.
 crons:
   drupal:
-    spec: '* * * * *'
+    spec: '*/5 * * * *'
```

Comment:
Set to 5 mins rather than every minute, because platform.sh only permits crons to run a minimum of every 5 minutes, even if you set it to less than that.